README.md

```diff
@@ -30,21 +34,25 @@ deposit: function(amount) {
 
 Any addition or subtraction expressions dealing with monetary amounts should protected with `Nat()` to guard against overflow/underflow errors. Without this check, the two balances might both be safe, but their sum might be too large to represent accurately, causing precision errors in subsequent computation:
 
-```
+```js
+const myOldBal = 12n; // BigInt numeric literal
+const amount = 3n;
 Nat(myOldBal + amount);
-const srcNewBal = Nat(srcOldBal - amount);
+
+const withdrawalAmount = 2n;
+const newBal = Nat(myOldBal - withdrawalAmount);
 ```
 
 ## Non-monetary usage
 
 Array indexes can be wrapped with `Nat()`, to guard against the surprising string coercion of non-integral index values:
 
-```
-const a = [2,4,6]
+```js
+const a = [2n, 4n, 6n];
 function add(index, value) {
   a[Nat(index)] = value;
 }
-add(3, 8); // works
+add(3n, 8n); // works
```

Comment:
Oh, you know I didn't look closely at this one. We're only applying `Nat` to the index, not the array values. So we should remove the `n` suffix from the `[2n, 4n, 6n]` line, and the call should be `add(3n, 8)`. It might make even more sense to explicitly use floats or strings as the values, so it's obvious which argument is supposed to be a Nat. Maybe have two example calls: `add(3n, "more"); // works: index is a Nat` and `add("more", 3n); // throws rather than add a key named "more"`

Reply:
I changed the values - let me know if that doesn't work!

Reply:
yep, looks good